gridcapa-task-manager-api/src/main/java/com/rte_france/farao/gridcapa_task_manager_api/entities/ProcessFile.java

```diff
@@ -4,7 +4,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-package com.farao_community.farao.gridcapa.task_manager.entities;
+package com.rte_france.farao.gridcapa_task_manager_api.entities;
```

Comment:
We must keep farao_community for the package it is the path for all the farao open source projects. Not a big fan of long 'gridcapa_task_manager_api' for a package name I would rather keep the old path.